integration_tests/mockApis/orchestration.ts

```diff
@@ -234,6 +235,34 @@ export default {
       },
     })
   },
+  stubGetVisitsWithoutSessionTemplate: ({
+    prisonId,
+    sessionDate,
+    visits,
+  }: {
+    prisonId: string
+    sessionDate: string
+    visits: VisitPreview[]
+  }): SuperAgentRequest => {
+    return stubFor({
+      request: {
+        method: 'GET',
+        urlPath: '/orchestration/visits/session-template',
+        queryParameters: {
+          prisonCode: { equalTo: prisonId },
+          sessionTemplateReference: { absent: true },
```

Comment:
Got stuck on this for a while because another mock for the same URL but with a different set of query params was getting matched. Adding the `absent` param means it **won't** match if that param is there - instead of the default of it being optional.